crates/db/store-sled/src/ol/schemas.rs

```diff
@@ -0,0 +1,41 @@
+use ssz::{Decode, Encode};
+use strata_db_types::traits::BlockStatus;
+use strata_identifiers::{OLBlockCommitment, OLBlockId};
+use strata_ol_chain_types_new::OLBlock;
+use typed_sled::codec::{CodecError, ValueCodec};
+
+use crate::{
+    define_table_with_default_codec, define_table_with_integer_key, define_table_without_codec,
+    impl_borsh_key_codec, impl_borsh_value_codec,
+};
+
+define_table_without_codec!(
+    /// A table to store OL Block data. Maps block commitment to Block
+    (OLBlockSchema) OLBlockCommitment => OLBlock
+);
+
+impl_borsh_key_codec!(OLBlockSchema, OLBlockCommitment);
+
+define_table_with_default_codec!(
+    /// A table to store OL Block status. Maps block commitment to BlockStatus
+    (OLBlockStatusSchema) OLBlockCommitment => BlockStatus
```

Comment:
We won't most likely have to iterate lexicographically on this. But to be safe, we can use `define_table_with_seek_key_codec!`.